toolchain/cc_toolchain_config.bzl

```diff
@@ -0,0 +1,151 @@
+    ACTION_NAMES.preprocess_assemble,
+    ACTION_NAMES.linkstamp_compile,
+    ACTION_NAMES.c_compile,
+    ACTION_NAMES.cpp_compile,
+    ACTION_NAMES.cpp_header_parsing,
+    ACTION_NAMES.cpp_module_codegen,
+    ACTION_NAMES.cpp_module_compile,
+    ACTION_NAMES.clif_match,
+    ACTION_NAMES.lto_backend,
+]
+
+def _impl(ctx):
+  tool_paths = [
+      tool_path(
+          name = "gcc",
+          path = "/usr/bin/clang",
+      ),
+      tool_path(
+          name = "ld",
+          path = "/usr/bin/ld",
+      ),
+      tool_path(
+          name = "ar",
+          path = "/usr/bin/llvm-ar-11",
+      ),
+      tool_path(
+          name = "compat-ld",
+          path = "/usr/bin/ld",
+      ),
+      tool_path(
+          name = "cpp",
+          path = "/usr/bin/clang-cpp",
+      ),
+      tool_path(
+          name = "dwp",
+          path = "/usr/bin/llvm-dwp",
+      ),
+      tool_path(
+          name = "gcov",
+          path = "/usr/bin/llvm-profdata",
+      ),
+      tool_path(
+          name = "nm",
+          path = "/usr/bin/llvm-nm",
+      ),
+      tool_path(
+          name = "objcopy",
+          path = "/usr/bin/llvm-objcopy",
+      ),
+      tool_path(
+          name = "objdump",
+          path = "/usr/bin/llvm-objdump",
+      ),
+      tool_path(
+          name = "strip",
+          path = "/usr/bin/llvm-strip",
+      ),
+  ]
+
+  features = [
+      feature(
+          name = "default_linker_flags",
+          enabled = True,
+          flag_sets = [
+              flag_set(
+                  actions = all_link_actions,
+                  flag_groups = ([
+                      flag_group(
+                          flags = [
+                              "-lstdc++",
+                              "--target=" + ctx.attr.target_full_name,
+                          ],
+                      ),
+                  ]),
+              ),
+          ],
+      ),
+  ]
+
+  features.append(feature(
+      name = "default_compile_flags",
+      enabled = True,
+      flag_sets = [
+          flag_set(
+              actions = all_compile_actions,
+              flag_groups = [
+                  flag_group(
+                      flags = [
+                          ctx.attr.bit_flag,
+                          "-Wall",
+                          "-no-canonical-prefixes",
+                          "--target=" + ctx.attr.target_full_name,
+                          "-isystem",
+                          ctx.attr.toolchain_dir,
+                          ctx.attr.include_flag,
+                      ],
+                  ),
+              ],
+          ),
+      ],
+  ))
+
+  return cc_common.create_cc_toolchain_config_info(
+      abi_libc_version = ctx.attr.target_cpu,
+      abi_version = ctx.attr.target_cpu,
+      ctx = ctx,
+      compiler = "clang",
+      cxx_builtin_include_directories = [
+          ctx.attr.toolchain_dir,
+          "/usr/include",
+          "/usr/lib/clang/11.0.1/include",
```

Comment:
Is there potentially a symlink somewhere that lets us avoid using explicit versions?

Reply:
I removed the version and it still works.